tests/php/Forms/HTMLEditor/HTMLEditorConfigTest/NullEditorConfig.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\Forms\Tests\HTMLEditor\HTMLEditorFieldTest;
+
+use SilverStripe\Forms\HTMLEditor\HTMLEditorConfig;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorRuleSet;
+
+class NullEditorConfig extends HTMLEditorConfig
```

Comment:
We probably don't need this?

Reply:
Conversation about this can be in https://github.com/silverstripe/silverstripe-framework/pull/11617#discussion_r1988380537